.github/workflows/build-test.yaml

```diff
@@ -708,7 +708,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s_version: [ v1.23.15-k3s1,v1.24.9-k3s2,v1.25.6-k3s1,v1.26.1-k3s1 ]
+        k8s_version: [ v1.26.1-k3s1 ] # there's no need to run this test on multiple k3s versions
```

Comment:
is this an intentional change? makes sense to only run the new test below on one version, but this one might be good to run on multiple supported versions

Reply:
```suggestion
        k8s_version: [ v1.23.15-k3s1,v1.24.9-k3s2,v1.25.6-k3s1,v1.26.1-k3s1 ]
```